### PR TITLE
Bug 1815552: Move to namespace openshift-cluster-storage-operator

### PR DESCRIPTION
**Pull request by jsafrane**
Move both the operand and the operator to this namespace.

### CODE REVIEW

File: pkg/operator/operator.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ var deploymentVersionHashKey = operatorv1.GroupName + "/rvs-hash"
 
 const (
 	targetName        = "csi-snapshot-controller"
-	targetNamespace   = "openshift-cluster-storage-controllers"
-	operatorNamespace = "openshift-cluster-storage-operators"
+	targetNamespace   = "openshift-cluster-storage-operator"
+	operatorNamespace = "openshift-cluster-storage-operator"
```
Comment (huffmanca) [inline pkg/operator/operator.go]: Should we consolidate these into a single variable?

Comment (jsafrane) [reply pkg/operator/operator.go]: I'd rather keep them separated, just in case someone wants to separate them again.

Comment (huffmanca) [reply pkg/operator/operator.go]: Sounds good to me.

**Comment by smarterclayton** [comment]
Architecture approval on naming granted

**Pull request merged**